Fixes for spanish movies

Ikax opened the pull request:
I have modified the search method in this provider to correct it. Right now it works well in PC but it is still too slow for the raspberry 2 (I'm still working on it).

Addiditonally in common.py I have modified the transation function to use it with TV Shows and I have added a function to remove accent marks that is used in a change I made in another provider (which I am going to after this one).

mancuniancol commented:
I don't understand why you change the request post for get?

The form from the web page is:
` <form id="searchform" method="post" id="frm_buscar" name="frm_buscar" action="http://www.newpct1.com/buscar" style="width:350px;height:44px;background:url(http://www.newpct1.com/pct1/library/content/template/images/search-box.png) no-repeat right;">
          <div id="content-search">
		  <input autocomplete="off"  maxlength="23" type="text" name="q" id="search" class="search-box" onblur="if(this.value==''){this.value='Buscar aqui..';}else{document.frm_buscar.submit();}" onfocus="if(this.value=='Buscar aqui..')this.value='';" value="Buscar aqui.." />
          <input type="submit" id="searchsubmit" class="search-button"/>
		  </div>
	  </form>`

As you can see, it is a Post request using q.  Your proposal, it could not work because it doesn't make the same requests than web site.

Second, why are you translating the tv shows? usually, they are not translated.

I will keep this line from your code:
unicode_name = unicode(name, 'utf-8', 'ignore')

I need to do some test with no latin language, though

mancuniancol commented:
Btw, I really appreciate your feedback

Ikax commented:
Hi, thanks for the prompt answer!

I changed the request for get because the searching engine of the webpage is quite bad honestly, it seems to search each word separately and returns a lot of incorrect links.

I tried to use the original request and add " before and after the title to improve the results but I could not manage to make it work like that, so I changed it so that I could add the encoded symbol manually (%22). It does work though, the webpage uses http://www.newpct1.com/index.php?page=buscar in other places, and thats when I thought of using that.

I was going to comment this in the upload to see whether you could make it work with the original request (which is a lot cleaner) but I forgot, sorry!

By the way, I use your english providers all the time! Thank you for work.
